src/cryptoadvance/specter/node.py

```diff
@@ -530,6 +531,40 @@ def check_blockheight(self):
     def is_liquid(self):
         return is_liquid(self.chain)
 
+    def delete_wallet_file(self, wallet) -> bool:
+        """Deleting the wallet file located on the node. This only works if the node is on the same machine as Specter.
+        Returns True if the wallet file could be deleted, otherwise returns False."""
+        wallet_rpc_path = os.path.join(
+            wallet.rpc_folder, wallet.alias
+        )  # e.g. specter/jade_wallet
+        logger.debug(f"The wallet_rpc_path is: {wallet_rpc_path}")
+        self.rpc.unloadwallet(wallet_rpc_path)
```

Comment:
The method doesn't sound like it's the responsibility of it to unload the wallet. So i assume it's doing that here just to be sure? What happens if it's already unloaded? So this is safe to call twice? Maybe make a comment here!